### PR TITLE
Patch relnotes.k8s.io to release v1.30.0-alpha.2

### PR DESCRIPTION
**Pull request by OrlinVasilev**
Automated patch to update relnotes.k8s.io to k/k version `v1.30.0-alpha.2`

### CODE REVIEW

File: src/environments/assets.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 export const assets = [
+  'assets/release-notes-1.30.0-alpha.2.json',
```
Comment (ramrodo) [inline src/environments/assets.ts:2]: @OrlinVasilev did you add this manually? or it was added automatically by krel?

**Review by ramrodo — COMMENTED** [review]
Question related to the changes of the assets file.

File: src/environments/assets.ts (hunk 1 above)
Comment (ramrodo) [reply src/environments/assets.ts:2]: Nvm, I reviewed [other PR](https://github.com/kubernetes-sigs/release-notes/pull/212/files) and this was added by `krel`.

**Review by ramrodo — APPROVED** [review]
/lgtm

**Review by puerco — APPROVED** [review]

**Pull request merged**